installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc

```diff
@@ -0,0 +1,41 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-azure-preparing-ipi"]
+= Preparing to install a cluster on Azure
+include::_attributes/common-attributes.adoc[]
+:context: installing-azure-preparing-ipi
+
+toc::[]
+
+To prepare for installation of an {product-title} cluster on {azure-short}, complete the following steps:
+
+* You have xref:../../../installing/installing-preparing.adoc#installing-preparing[selected a cluster installation method].
+
+* You xref:../../../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[configured an Azure account] to host the cluster and determined the tested and validated region to deploy the cluster to.
+
+* If you use a firewall, you have xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
```

Comment:
```suggestion
* If you use a firewall, you xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
```

Reply:
As per [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites), this is a valid usage.